content/en/tracing/setup_overview/open_standards/otlp_ingest_in_the_agent.md

```diff
@@ -0,0 +1,233 @@
+---
+title: OTLP Trace Ingestion by the Datadog Agent
+kind: documentation
+description: 'Ingest OTLP trace data through the Datadog Agent'
+further_reading:
+aliases:
+---
+
+
+OTLP Ingest in the Agent is a way to send telemetry data directly from [OpenTelemetry SDKs][1] to Datadog Agent. Since versions 6.32.0 and 7.32.0, the Datadog Agent can ingest OTLP traces and [OTLP metrics][2] through gRPC or HTTP.
```

Comment:
```suggestion
OTLP Ingest in the Agent is a way to send telemetry data directly from applications instrumented with [OpenTelemetry SDKs][1] to Datadog Agent. Since versions 6.32.0 and 7.32.0, the Datadog Agent can ingest OTLP traces and [OTLP metrics][2] through gRPC or HTTP.
```